tractor/_actor.py

```diff
@@ -694,6 +717,10 @@ async def _async_main(
             # or completed
             self.cancel_server()
 
+            # teardown msg loop with parent
+            if self._parent_chan_cs:
```

Comment:
This is new to shut down logic: we always keep the parent channel message loop running until the very end.

It is cancelled as the last thing before process termination.